Telerik.Sitefinity.Frontend.EmailCampaigns/Mvc/Models/UnsubscribeForm/UnsubscribeFormModel.cs

```diff
@@ -242,20 +254,30 @@ private void Unsubscribe(NewslettersManager newslettersManager, Subscriber subsc
         /// </summary>
         /// <param name="subscriber">The subscriber.</param>
         /// <param name="issue">The issue.</param>
+        /// <param name="list">The mailing list.</param>
         /// <returns></returns>
-        private string GetUnsubscribeSuccessfulMessage(Subscriber subscriber, Campaign issue)
+        private string GetUnsubscribeSuccessfulMessage(Subscriber subscriber, Campaign issue, MailingList list)
```

Comment:
We can pass just the list here and have the check from outside in the diffrent cases